src/components/pages/order/OrderPage.js

```diff
@@ -1,17 +1,34 @@
-import { useNavigate, useLocation } from "react-router-dom"
+import { useLocation } from "react-router-dom"
+import styled from "styled-components"
+import { theme } from "../../../theme"
```

Comment:
```suggestion
import { { colors, borderRadius } } from "../../../theme"
```

Eventuellement, tu peux faire ça, mais en vrai, c'est pas très clean 😬